Jenkinsfile

```diff
@@ -524,6 +524,7 @@ def e2e(Map args = [:]) {
     def goVersionForE2E = readFile('.go-version').trim()
     withEnv(["GO_VERSION=${goVersionForE2E}",
               "BEATS_LOCAL_PATH=${env.WORKSPACE}/${env.BASE_DIR}",
+              "BEAT_VERSION=${env.VERSION}-SNAPSHOT",
```

Comment:
Umm, `triggerE2ETests` also appends `-SNAPSHOT`, do we have the risk of appending it twice?

Reply:
No, there is no risk, as we set that version to be used as input parameter of the downstream job for the e2e tests